tensilelite/Tensile/Tests/common/gemm/dtv.yaml

```diff
@@ -423,19 +423,22 @@ BenchmarkProblems:
         - KernelLanguage: ["Assembly"]
       ForkParameters:
         - MatrixInstruction:
-          - [16, 16, 16, 1,  1,   2, 1,  4,1 ]
+          - [16, 16, 16, 1,  1,   1, 1,  4,1 ] # MT = 64x16. Case to check kernel writer works
+          - [16, 16, 16, 1,  1,   2, 1,  4,1 ] # MT = 128x16. Check kernel works without vgprValuA
           - [16, 16, 16, 1,  1,   4, 1,  4,1 ]
+          - [16, 16, 16, 1,  1,   8, 1,  4,1 ] # MT = 512x16. Case to check LDS usage of Epilogues
+          - [16, 16, 16, 1,  1,   5, 1,  2,1 ] # MT = 160x16. Case to check kernel writer works
           - [32, 32,  8, 1,  1,   2, 1,  4,1 ]
           - [32, 32,  8, 1,  1,   4, 1,  4,1 ]
```

Comment:
1. The first problem set case shows the validation failure when LDS-of-AB < LDS-of-epilogue.
2. The second problem set case shows the unexpected appearance of [vgprValuA]_XXXXX: [vpgrValueA]s are not really storing any data from global memory, but they are still used to do mfma
3. The thrid problem set case shows the failures of kernel generation: The config doesn't even work, it says KeyError: 'vgprValuA_X0_I0'. Since it uses the "un-declared vgprValuA_XXXX" in somewhere later.